src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1150,7 +1162,14 @@ def execute(
             )
             .pipe(self.ag_fraction_owned)
             .assign(plant_part=self.part_name)
-            .pipe(self.add_install_year, gens_mega)
+            .pipe(
+                AddMaxMinAttribute("installation_year", self.part_name).execute,
```

Comment:
I'm sorry for throwing in a late potential monkey wrench but it miiiight make more sense to add these into `MakePlantParts.execute()` like the other attributes. Or at least standardizing where the attributes are added in. (you can search for ` for attribute_col in CONSISTENT_ATTRIBUTE_COLS:` to find the little section where the other attributes are added).

I'm not sure which place would feel cleaner, but I am tempted to think doing this in `MakePlantParts` makes more sense because the `PlantPart` doesn't actually need to know about the attributes. I'm not totally sure why I put this in here originally.

Reply:
Agree it makes more sense to add with the other attributes now that it's part of `AddAttribute`. Made this change.